assets/js/reservations.js

```diff
@@ -0,0 +1,28 @@
+window.viewFunctions = window.viewFunctions || {};
+viewFunctions.ajax_reservation = function(csrf_token, request_url) {
```

Comment:
Identifier 'ajax_reservation' is not in camel case.<br>Identifier 'csrf_token' is not in camel case.<br>Identifier 'request_url' is not in camel case.<br>'viewFunctions' is not defined.